### PR TITLE
Incidents management tool updates

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Updates for the Incidents management GA

-[Incident management](https://docs-staging.datadoghq.com/kaylyn/incidents-ga/monitors/incident_management/) preview
-[Datadog Clipboard](https://docs-staging.datadoghq.com/kaylyn/incidents-ga/monitors/incident_management/datadog_clipboard) preview

### CODE REVIEW

File: content/en/monitors/incident_management.md [hunk 1]
```diff
@@ -7,8 +7,6 @@ description: Create and manage incidents
 
 {{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
 
-## Overview
-
 Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:10]: Datadog’s Incident Management product provides a system through which your organization can effectively identify and mitigate incidents.

File: content/en/monitors/incident_management.md [hunk 2]
```diff
@@ -7,8 +7,6 @@ description: Create and manage incidents
 
 {{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}
 
-## Overview
-
 Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 
 
 Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:12]: Incidents and outages can be managed in Datadog alongside the metrics, traces, logs, and other data you are collecting. You can view and filter incidents that are relevant to you and your team.

File: content/en/monitors/incident_management.md [hunk 3]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
 * You do not know whether you should call an incident. Notify other people and increase severity appropriately.
 
-<div class="alert alert-warning">
-<a href="https://app.datadoghq.com/incidents">Incident Management</a> is currently in beta and available to all customers. For more information, reach out to sales@datadoghq.com.
-</div>
-
 ## Usage
 
-Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page for a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents.  You can configure additional fields that appear for all incidents in [Incident Settings][2].
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:21]: all ongoing and resolved incidents.

File: content/en/monitors/incident_management.md [hunk 4]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
 * An issue is or may be impacting customers or services.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:16]: An issue is or may be impacting customers or internal services.

File: content/en/monitors/incident_management.md [hunk 5]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
 * You do not know whether you should call an incident. Notify other people and increase severity appropriately.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:17]: Remove - Notify other people and increase severity appropriately.

File: content/en/monitors/incident_management.md (hunk 3 above)
Comment (cswatt) [inline content/en/monitors/incident_management.md:21]:
```suggestion
Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
```

File: content/en/monitors/incident_management.md [hunk 6]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
-You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}
+
+#### From the clipboard
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:33]: Use the Datadog Clipboard to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select **Open Clipboard**. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID. 

Note: the Datadog Clipboard is in beta

We should also link to docs

File: content/en/monitors/incident_management/_index.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor" style="width:80%;">}}
+
+#### From the incidents page
+
+In the [incidents UI][1], click the **New Incident** button to create an incident.
+
+#### From Slack
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management/_index.md:51]: From the Datadog Slack App

File: content/en/monitors/incident_management/_index.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+Once you have the [Datadog integration enabled on Slack][3], you can use `/datadog` to declare a new incident from any channel.
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management/_index.md:53]: Once you have the [Datadog integration enabled on Slack][3], from any Slack channel you can use the slash command `/datadog incident` to declare a new incident. 

In the creation modal, you add a descriptive title, select whether customers were impacted (yes, no, or unknown) and select a severity level (1-5, unknown). 

If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account, then the IC is assigned to a generic `Slack app user` and can be assigned to another IC in-app.

Read more about using the Datadog Slack App here.

File: content/en/monitors/incident_management.md [hunk 9]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
 ### Creating an incident
 
-You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}
+
+#### From the clipboard
```
Comment (cswatt) [inline content/en/monitors/incident_management.md:31]:
```suggestion
#### From the Clipboard
```

get it together

File: content/en/monitors/incident_management/_index.md (hunk 8 above)
Comment (meghanelizabeth) [reply content/en/monitors/incident_management/_index.md:53]: Updated with Natalie's comments in the draft doc

File: content/en/monitors/incident_management.md (hunk 6 above)
Comment (cswatt) [inline content/en/monitors/incident_management.md:33]:
```suggestion
Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the Clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the Clipboard and then click **Add to New Incident**. Everything on the Clipboard is added to the incident as a signal.
```

File: content/en/monitors/incident_management.md [hunk 10]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.
+
+{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard">}}
+
+{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard">}}
+
+**Note**: Data on the clipboard can also be exported to a new dashboard or a notebook in addition to an incident.
```
Comment (cswatt) [inline content/en/monitors/incident_management.md:39]:
```suggestion
**Note**: Data on the Clipboard can also be exported to a new dashboard or a notebook in addition to an incident.
```

File: content/en/monitors/incident_management.md [hunk 11]
```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
...
+
+Once you have the [Datadog integration enabled on Slack][3], you can use `/datadog` to declare a new incident from any channel.
+
+{{< img src="monitors/incidents/from-slack.png" alt="Create in incident from Slack">}}
+
+If the user declaring the incident is a part of your Datadog account, then that user becomes the Incident Commander (IC) by default. If the person declaring an incident is not part of your Datadog account, then the IC is assigned to a generic `Slack app user`. The IC can be changed on the [incidents page][1] in the Datadog app.
+
+Once you declare an incident from Slack, it generates an incident channel.
+
+For more information about the Datadog Slack integration, check out [the docs][3].
```
Comment (cswatt) [inline content/en/monitors/incident_management.md:61]:
```suggestion
For more information about the Datadog Slack integration, see [the documentation][3].
```

File: content/en/monitors/incident_management/_index.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+
+### Creating an incident
+
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph" style="width:80%;">}}
+
+#### From the clipboard
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:31]:
```suggestion
#### From the Clipboard
```

File: content/en/monitors/incident_management/_index.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:33]:
```suggestion
Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the Clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the Clipboard and then click **Add to New Incident**. Everything on the Clipboard is added to the incident as a signal.
```

File: content/en/monitors/incident_management/_index.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+
+{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard" style="width:80%;">}}
+
+{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard" style="width:80%;">}}
+
+**Note**: In addition to exporting from an incident, data on the clipboard can be exported to a new dashboard or a notebook.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:39]:
```suggestion
**Note**: In addition to exporting from an incident, data on the Clipboard can be exported to a new dashboard or a notebook.
```

File: content/en/monitors/incident_management/_index.md [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+
+Click on “Declare Incident” to finish creating your incident.
+
+### Updating the incident
+
+An incident’s status can be updated directly in the [overview page][1] of the incident or from Slack within the dedicated incident channel. To update an incident from Slack, use this slash command to pull up the update modal: `/datadog incident update`
+
+You can also update the impact section to specify if there was customer impact, the incident timeline, and whether or not it’s still active. This section also requires a description of the scope of impact to be completed.
+
+In the incident header, you can see the state, severity, timestamp, and impact duration of the incident, and who has responded to the incident. You can also notify responders of updates. There are quick links to chat channels, video conferencing, and attached postmortem (if one has been added).
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:96]:
```suggestion
In the incident header, you can see the incident's state, severity, timestamp, impact, and duration, as well as who has responded to the incident. You can also notify responders of updates. There are quick links to chat channels, video conferencing, and attached postmortem (if one has been added).
```

File: content/en/monitors/incident_management/_index.md [hunk 16]
```diff
@@ -0,0 +1,179 @@
...
+#### Status levels
+
+The default includes the statuses **Active**, **Stable**, and **Resolved**. **Completed** can be enabled or disabled. You can customize the description of each status level to fit the requirements of your organization. 
+
+* Active: Incident affecting others.
+* Stable: Incident no longer affecting others, but investigations incomplete.
+* Resolved: Incident no longer affecting others and investigations complete.
+* Completed: All remediation complete.
+
+#### Assessment Fields
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:107]:
```suggestion
#### Assessment fields
```

File: content/en/monitors/incident_management/_index.md [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+
+Assessment fields are the metadata and context that you can define per incident. These fields are [key:value metric tags][4]. These field keys are added in settings, and the values are then available when you are assessing the impact of an incident on the overview page. For example, you can add an Application field. The following fields are available for assessment in all incidents:
+
+* **Root Cause**: This text field allows you to enter the description of the root cause, triggers, and contributing factors of the incident. 
+* **Detection Method**: Specify how the incident was detected via these default options: customer, employee, monitor, other or unknown.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:112]:
```suggestion
* **Detection Method**: Specify how the incident was detected with these default options: customer, employee, monitor, other, or unknown. 
```

File: content/en/monitors/incident_management/_index.md [hunk 18]
```diff
@@ -0,0 +1,179 @@
...
+* Resolved: Incident no longer affecting others and investigations complete.
+* Completed: All remediation complete.
+
+#### Assessment Fields
+
+Assessment fields are the metadata and context that you can define per incident. These fields are [key:value metric tags][4]. These field keys are added in settings, and the values are then available when you are assessing the impact of an incident on the overview page. For example, you can add an Application field. The following fields are available for assessment in all incidents:
+
+* **Root Cause**: This text field allows you to enter the description of the root cause, triggers, and contributing factors of the incident. 
+* **Detection Method**: Specify how the incident was detected via these default options: customer, employee, monitor, other or unknown. 
+* **Services**: If you have APM configured, your APM services will be available for incident assessment. To learn more about configuring your services in APM, see [the docs][5].
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:113]:
```suggestion
* **Services**: If you have APM configured, your APM services are available for incident assessment. To learn more about configuring your services in APM, see [the docs][5]. 
```

File: content/en/monitors/incident_management/_index.md [hunk 19]
```diff
@@ -0,0 +1,179 @@
...
+* Completed: All remediation complete.
+
+#### Assessment Fields
+
+Assessment fields are the metadata and context that you can define per incident. These fields are [key:value metric tags][4]. These field keys are added in settings, and the values are then available when you are assessing the impact of an incident on the overview page. For example, you can add an Application field. The following fields are available for assessment in all incidents:
+
+* **Root Cause**: This text field allows you to enter the description of the root cause, triggers, and contributing factors of the incident. 
+* **Detection Method**: Specify how the incident was detected via these default options: customer, employee, monitor, other or unknown. 
+* **Services**: If you have APM configured, your APM services will be available for incident assessment. To learn more about configuring your services in APM, see [the docs][5]. 
+    * If you are not using Datadog APM, you can upload service names via CSV. Any values uploaded via CSV will only be available within Incident Management for incident assessment purposes.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:114]:
```suggestion
    * If you are not using Datadog APM, you can upload service names as a CSV. Any values uploaded via CSV are only be available within Incident Management for incident assessment purposes.
```

File: content/en/monitors/incident_management/_index.md [hunk 20]
```diff
@@ -0,0 +1,179 @@
...
+    * Datadog deduplicates service names case-insensitively,, so if you use "My Service" or "my service", only the manually added one is shown.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:115]:
```suggestion
    * Datadog deduplicates service names case-insensitively, so if you use "My Service" or "my service", only the manually added one is shown.
```

File: content/en/monitors/incident_management/_index.md [hunk 21]
```diff
@@ -0,0 +1,179 @@
...
+    * Datadog overrides APM service names in favor of the manually uploaded list. 
+    * Note that if the service is an APM service and no metrics are posted in the past seven days, it does not appear in the search results.
+    * Further integrate with Datadog products and accurately assess service impact. The Services property field is automatically populated with APM services for customers using Datadog APM 
+* **Teams**: Populated from a CSV upload. Any values uploaded via CSV will only be available within Incident Management for incident assessment purposes.
```
Comment (cswatt) [inline content/en/monitors/incident_management/_index.md:119]:
```suggestion
* **Teams**: Populated from a CSV upload. Any values uploaded via CSV are only available within Incident Management for incident assessment purposes.
```

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+kind: documentation
+disable_toc: true
+description: Create and manage incidents
+---
+
+**The Datadog Clipboard is currently in public beta**
+
+# Overview
+
+The Datadog Clipboard is a cross-platform tool for collecting and sharing signals across contexts. It is personal to each user and stores all copied graphs alongside any saved links. Signals can be grouped and exported to a Dashboard, Notebook, or Incident.
```
Comment (cswatt) [inline content/en/monitors/incident_management/datadog_clipboard.md:12]:
```suggestion
The Datadog Clipboard is a cross-platform tool for collecting and sharing signals across contexts. It is personal to each user and stores all copied graphs alongside any saved links. Signals can be grouped and exported to a dashboard, notebook, or incident.
```

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+
+The Clipboard works on all pages in Datadog and keeps a record of all graphs copied by an individual user. The Clipboard does not automatically copy query text, event JSON, or other text-based content. 
+
+## Opening the Clipboard
+
+To open the Clipboard, copy any graph and click “Open Clipboard” in the toast.
+
+{{< img src="monitors/incidents/open-clipboard.png" alt="Open a graph in the Clipboard"  style="width:80%;">}}
+
+Or, click “`Cmd + Shift + X` to open” on the minimized Clipboard.
```
Comment (cswatt) [inline content/en/monitors/incident_management/datadog_clipboard.md:26]:
```suggestion
Or, click “`Cmd/Ctrl + Shift + X` to open” on the minimized Clipboard. 
```

windows users

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+
+## Opening the Clipboard
+
+To open the Clipboard, copy any graph and click “Open Clipboard” in the toast.
+
+{{< img src="monitors/incidents/open-clipboard.png" alt="Open a graph in the Clipboard"  style="width:80%;">}}
+
+Or, click “`Cmd + Shift + X` to open” on the minimized Clipboard. 
+
+The Clipboard can also be opened and closed using `Cmd + Shift + X`. To minimize the Clipboard, click the Minimize icon. The minimized Clipboard persists on all pages of Datadog.
```
Comment (cswatt) [inline content/en/monitors/incident_management/datadog_clipboard.md:28]:
```suggestion
The Clipboard can also be opened and closed using `Cmd/Ctrl + Shift + X`. To minimize the Clipboard, click the Minimize icon. The minimized Clipboard persists on all pages of Datadog.
```

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+
+## Adding Clips
+
+To add a graph, copy it with `Cmd+C` or click “Copy” in the export menu. Once the Clipboard is open, copied graphs get added automatically.
```
Comment (cswatt) [inline content/en/monitors/incident_management/datadog_clipboard.md:32]:
```suggestion
To add a graph, copy it with `Cmd/Ctrl + C` or click “Copy” in the export menu. Once the Clipboard is open, copied graphs get added automatically. 
```

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 26]
```diff
@@ -0,0 +1,54 @@
...
+
+Each item in the Clipboard can be opened, cloned, or deleted; these options are available when you hover over any signal. Opening an item navigates to the link of the original signal. Open the source of any graph (like the dashboard it was clipped from) by clicking the title at the bottom of the item.
+
+{{< img src="monitors/incidents/managing-clips.png" alt="Manage your clips"  style="width:80%;">}}
+
+The Clipboard holds a maximum of 20 graphs. Remove graphs by deleting them individually, or by clicking “Clear Clipboard” on the bottom left. If more than 20 graphs are added, the oldest graphs, stored furthest to the left, are removed automatically.
+
+## Exporting
+
+Items on the Clipboard can be exported to Dashboards, Notebooks, or Incidents. `Shift + Click` to select multiple items. In the export menu, choose a new export destination, or search through existing Dashboards, Notebooks, and Incidents.
```
Comment (cswatt) [inline content/en/monitors/incident_management/datadog_clipboard.md:48]:
```suggestion
Items on the Clipboard can be exported to dashboards, notebooks, or incidents. `Shift + Click` to select multiple items. In the export menu, choose a new export destination, or search through existing dashboards, notebooks, and incidents. 
```

File: content/en/monitors/incident_management.md (hunk 10 above)
Comment (niustephanie) [reply content/en/monitors/incident_management.md:39]: Data on the Clipboard can also be exported to a new or existing dashboard or notebook in addition to an incident.

File: content/en/monitors/incident_management/datadog_clipboard.md [hunk 27]
```diff
@@ -0,0 +1,54 @@
...
+description: Create and manage incidents
+---
+
+**The Datadog Clipboard is currently in public beta**
+
+# Overview
+
+The Datadog Clipboard is a cross-platform tool for collecting and sharing signals across contexts. It is personal to each user and stores all copied graphs alongside any saved links. Signals can be grouped and exported to a Dashboard, Notebook, or Incident.
+
+{{< img src="monitors/incidents/clipboard-main.png" alt="The main clipboard">}}
```
Comment (niustephanie) [inline content/en/monitors/incident_management/datadog_clipboard.md:14]: image is slightly cropped- can use this one https://a.cl.ly/E0u4GZmR

File: content/en/monitors/incident_management/_index.md (hunk 8 above)
Comment (apigirl) [reply content/en/monitors/incident_management/_index.md:53]: completely missed these, thanks!

File: content/en/monitors/incident_management.md [hunk 28]
```diff
@@ -7,30 +7,64 @@ description: Create and manage incidents
...
-You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}
+
+#### From the Clipboard
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the Clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the Clipboard and then click **Add to New Incident**. Everything on the Clipboard is added to the incident as a signal.
```
Comment (apigirl) [inline content/en/monitors/incident_management.md:33]:
```suggestion
Use the beta Datadog Clipboard to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select Open Clipboard. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click Add to New Incident. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID.
```

File: content/en/monitors/incident_management.md [hunk 29]
```diff
@@ -7,30 +7,66 @@ description: Create and manage incidents
...
+
+{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard">}}
+
+**Note**: Data on the Clipboard can also be exported to a new or existing dashboard or a notebook in addition to an incident.
+
+#### From a monitor
+
+You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor">}}
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:47]: Screen Shot 2020-11-19 at 2 10 37 PM

These can also be added to existing incidents

File: content/en/monitors/incident_management/_index.md [hunk 30]
```diff
@@ -0,0 +1,185 @@
...
+
+Click on “Declare Incident” to finish creating your incident.
+
+### Updating the incident and the incident timeline
+
+An incident’s status can be updated directly in the [overview page][1] of the incident or from Slack within the dedicated incident channel. To update an incident from Slack, use this slash command to pull up the update modal: `/datadog incident update`
+
+You can also update the impact section to specify if there was customer impact, the incident timeline, and whether or not it’s still active. This section also requires a description of the scope of impact to be completed.
+
+In the incident header, you can see the incident's state, severity, timestamp, impact, and duration, as well as who has responded to the incident. You can also notify responders of updates. There are quick links to chat channels, video conferencing, and attached postmortem (if one has been added).
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management/_index.md:102]: There are quick links to add and access chat channels (if not using the Datadog Slack App), video conferencing,

File: content/en/monitors/incident_management.md [hunk 31]
```diff
@@ -51,29 +87,35 @@ You can declare an incident directly from a graph. You can also go to the upper
...
 {{< img src="monitors/incidents/workflow-2-modal-1.png" alt="New Incident"  style="width:60%;">}}
 
-#### 3. Communicate and begin troubleshooting
+### 3. Communicate and begin troubleshooting
 
 If you have the [Datadog Slack app][3] installed, the Slack integration can automatically create a new channel dedicated to the incident, so you can consolidate communication with your team and begin troubleshooting.
 
 For non-EU customers who use Slack, [sign up for beta access][4] to the Datadog Slack app. For EU customers who use Slack, stay informed about the Slack app by emailing support@datadoghq.com.
 
 {{< img src="monitors/incidents/workflow-3-slack-1.png" alt="Communicate"  style="width:80%;">}}
```
Comment (meghanelizabeth) [inline content/en/monitors/incident_management.md:116]: Screen Shot 2020-11-05 at 9 07 40 PM

Comment (meghanelizabeth) [reply content/en/monitors/incident_management.md:116]: image replacement

**Review by cswatt — APPROVED** [review]

**Pull request merged**